handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
 };
 
 Character.prototype = Object.create(Player.prototype);
 Character.prototype.constructor = Character;
 
+Character.prototype.addItem = function(newItem) {
+    for (var item in this.inventory) {
+        if (this.inventory[item].name === newItem.name) {
+            this.inventory[item].quantity += newItem.quantity;
+            return "Added new quantity to existing quantity.";
+        }
+    }
+
+    this.inventory.push(newItem);
+    return "Added new item to inventory";
+}
+
+Character.prototype.removeItem = function(targetItem) {
+    for (var item in this.inventory) {
+        if (this.inventory[item].name === targetItem.name) {
+            this.inventory[item].quantity -= targetItem.quantity;
+            if (this.inventory[item].quantity === 0) {
+                this.inventory.splice(item, 1);
+            }
+            return "Removed quantity from existing quantity.";
+        }
+    }
+
+    return "Item not found in inventory";
+}
+
+Character.prototype.listInventory = function() {
+    for (var item in this.inventory) {
+        console.log(this.inventory[item].name + ", " + 
```

Comment:
In console.log, you don't need to append the strings together, you can just comma separate them and it'll combine them together. (Bonus is that if you console.log an object, it'll print the object in a nicer format than if you stringify it.